ClosedXML.Tests/Examples/TablesTests.cs

```diff
@@ -10,27 +10,19 @@ public class TablesTests
         [Test]
         public void InsertingTables()
         {
-            var allowedDiff = "/xl/worksheets/sheet1.xml :NonEqual\n";
-
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                allowedDiff = null;
-            }
-
-            TestHelper.RunTestExample<InsertingTables>(@"Tables\InsertingTables.xlsx", false, allowedDiff);
+            TestHelper.RunTestExample<InsertingTables>(@"Tables\InsertingTables.xlsx", false, null, true);
```

Comment:
shouldn't that also check the columns on Windows? => ignoreColumnFormats: !RuntimeInformation.IsOSPlatform(OSPlatform.Windows)